src/subcommand/giraffe_main.cpp

```diff
@@ -387,6 +389,7 @@ int main_giraffe(int argc, char** argv) {
     #define OPT_REF_PATHS 1010
     #define OPT_SHOW_WORK 1011
     #define OPT_NAMED_COORDINATES 1012
+    #define OPT_SHOW_EXCLUDE_OVERLAPPING_MIN 1013
```

Comment:
```suggestion
    #define OPT_EXCLUDE_OVERLAPPING_MIN 1013
```
There's no reason to put "SHOW" in here.